### ISSUE TITLE
the only thing that works is the current temperature

### ISSUE THREAD

**Issue by Santanachia** [tetele/hvac_group#58]
## System Information

version | core-2024.1.5
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.6
os_name | Linux
os_version | 6.1.63-haos-raspi
arch | aarch64
timezone | Europe/Warsaw
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4958
Installed Version | 1.34.0
Stage | running
Available Repositories | 1385
Downloaded Repositories | 23
HACS Data | ok

</details>

<details><summary>Airly</summary>

can_reach_server | ok
-- | --
requests_remaining | 58
requests_per_day | 100

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.4
-- | --
update_channel | stable
supervisor_version | supervisor-2023.12.1
agent_version | 1.6.0
docker_version | 24.0.7
disk_total | 469.4 GB
disk_used | 17.6 GB
healthy | true
supported | true
board | rpi3-64
supervisor_api | ok
version_api | ok
installed_addons | Advanced SSH & Web Terminal (17.0.5), File editor (5.7.0), Samba NAS (12.2.0-nas1), HassOS SSH port 22222 Configurator (0.9.3), SQLite Web (4.1.0), Network UPS Tools (0.13.0), Home Assistant Google Drive Backup (0.112.1), eWeLink Smart Home (1.4.3), ESPHome (2023.12.9), Cloudflared (5.1.1), MariaDB (2.6.1), phpMyAdmin (0.9.1)

</details>

<details><summary>Husqvarna Automower</summary>

can_reach_server | ok
-- | --

</details>

<details><summary>Dashboards</summary>

dashboards | 3
-- | --
resources | 11
views | 9
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | January 25, 2024 at 09:18
-- | --
current_recorder_run | February 4, 2024 at 10:42
estimated_db_size | 94.70 MiB
database_engine | mysql
database_version | 10.6.12

</details>

<details><summary>Sonoff</summary>

version | 3.5.4 (a4a8c5f)
-- | --
cloud_online | 1 / 1
local_online | 1 / 1

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/tetele/hvac_group/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

The newly created helper does not show the current status of the connected heaters, nor the set temperature. It also does not allow you to set the temperature.

### Reproduction steps

1. create a group from two heaters

### Debug logs

```text
This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/hvac_group/climate.py:935
Integration: HVAC Group (documentation, issues)
First occurred: 10:53:19 (3 occurrences)
Last logged: 11:15:06

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/hvac_group/climate.py", line 800, in async_sensor_state_changed_listener
    await self.async_update_temperature_sensor(
  File "/config/custom_components/hvac_group/climate.py", line 935, in async_update_temperature_sensor
    float(new_temperature) if new_temperature is not None else new_temperature
    ^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'unavailable'
```

**Comment by Santanachia** [comment]
the same source of the problem as in https://github.com/tetele/hvac_group/issues/51

**Comment by tetele** [comment]
It's not quite the same issue

**Comment by tetele** [comment]
Should be fixed in [0.2.0-beta.2](https://github.com/tetele/hvac_group/releases/tag/0.2.0-beta.2) - it will eliminate the error message and make the group entity `unavailable` if the temperature sensor was never available.
